src/commands/fun/diceroll.ts

```diff
@@ -0,0 +1,35 @@
+import { Message } from 'discord.js';
+import { Command } from '@lib/types/Command';
+
+const DEFAULT_RANGE = [1, 6];
+export default class extends Command {
+
+	description = `Get a random integer within a user-specified range (min and max inclusive). If no range is specified, defaults to a range from ${DEFAULT_RANGE[0]} to ${DEFAULT_RANGE[1]}.`;
+	usage = '[min #] | [max #]';
+	aliases = ['random', 'rand'];
```

Comment:
This is mildly unimportant but you might want to consider also having aliases somewhat similar to the actual command name (but shortened), like s;dice or s;roll.